src/PJ_lsat.c

```diff
@@ -61,8 +61,11 @@ static XY e_forward (LP lp, PJ *P) {          /* Ellipsoidal, forward */
         sav = lampp;
         lamtp = lp.lam + Q->p22 * lampp;
         cl = cos(lamtp);
+#ifdef fixme
+        /* lamtp is not read any more */
```

Comment:
Has `lamtp` been used previously or is this just something that has been left in by mistake when originally implementing this projection?

Reply:
History ( https://github.com/OSGeo/proj.4/blame/4.8.0/src/PJ_lsat.c ) shows that this issues dates back to the initial commit 19 years ago...

Reply:
I think it is safe to remove those lines then :-) Same for the one below.